libs/blocks/modal/modal.js

```diff
@@ -16,10 +17,24 @@ export function findDetails(hash, el) {
   return { id, path, isHash: hash === window.location.hash };
 }
 
+export function sendAnalytics(event) {
+  // eslint-disable-next-line no-underscore-dangle
+  window._satellite?.track('event', {
+    xdm: {},
+    data: {
+      web: { webInteraction: { name: event.type } },
+      _adobe_corpnew: { digitalData: event.data },
+    },
+  });
+}
+
 function closeModal(modal) {
   const { id } = modal;
-  const closeEvent = new Event('milo:modal:closed');
```

Comment:
Renaming this will require consumers to update their implementations, this event is currently used in BACOM, Express and the footer, as can be seen [here](https://github.com/search?q=org%3Aadobecom%20milo%3Amodal%3Aclosed&type=code).

Is there any way we can keep the event name the same? Otherwise, a cross-team sync will be needed to implement this change as soon as it is merged in.

Reply:
I have updated the event name based on the live pages. You can verify that here https://www.adobe.com/ru (click close of georouting modal).
I'll ask this question in #milo-community to get confirmation from Benjamin and Uma

Reply:
I'd probably stick to the JS event `milo:modal:closed` and then a different name for the Analytics event

Reply:
Thanks for the suggestion @mokimo . I have incorporated it in my code now.